boa/src/exec/mod.rs

```diff
@@ -741,7 +743,9 @@ impl Interpreter {
             _ => input.clone(),
         }
     }
+
     /// to_string() converts a value into a String
```

Comment:
This should either be:
```suggestion
    /// `to_string()` converts a value into a `String`.
```
 or:

```suggestion
    /// Converts a value into a `String`.
```